airlock/views.py

```diff
@@ -210,8 +227,9 @@ def request_view(request, request_id: str, path: str = ""):
     )
 
     context = {
-        "workspace": release_request.workspace,
+        "workspace": api.get_workspace(release_request.workspace),
```

Comment:
I fixed this too, in a PR on Friday. I guess it can be closed in favour of this one

Reply:
ah, yeah, I approved it I think. This was just a drive by, lets merge yours